### ISSUE TITLE
Prompts for password on startup, works whether or not I enter it

### ISSUE THREAD

**Issue by cvandesande** [BelledonneCommunications/linphone-desktop#272]
Hi, I checked the other issues but mine seems different.

Using an external Asterisk 16.2.0 server, on TLS.  Using Linphone Desktop/4.1.1-655-g95245907.

When I open Linphone desktop, I immediately get prompted to enter my password ("Unable to Authenticate"). However if I wait a couple of seconds, I can see the spinner turn green on my account.
At this point I can either re-enter my password, or press cancel, it makes no difference, I can still make calls.

Any ideas? I wasn't sure exactly how to get the desktop logs, but I ran Linphone on verbose mode and attached the output.

[linphone-verbose.log](https://github.com/BelledonneCommunications/linphone-desktop/files/2891712/linphone-verbose.log)

**Comment by cvandesande** [comment]
I've looked at this more.
It looks like the first SIP REGISTER command doesn't include the realm information, so Asterisk sends a SIP 403 unauthorized.  In the SIP 403, the realm is mentioned, so Linphone sends a second REGISTER command, this time including the realm information, and succeeds.

**Comment by hartois** [comment]
it seems to me I have the same problem:
![image](https://user-images.githubusercontent.com/11641656/100373743-4b91dc80-301c-11eb-9e03-9e7b70c85eee.png)

Why Linphone send first REGISTER without realm?

**Comment by hartois** [comment]
![image](https://user-images.githubusercontent.com/11641656/100434393-79733180-30ad-11eb-9182-dadcf7522bb3.png)

As I understand it - Unauthorized packet is the correct part of the authentication process.

**Comment by hartois** [comment]
In my case:
Desktop 4.2.4 - Qt5.12.5
Core 4.4.9-1-g18c59c7db

**Comment by cvandesande** [comment]
I agree this issue isn't fixed.  See screenshot, app has successfully authenticated despite asking for my password.  Tested with 4.2.4 on Debian using appimage.
![ksnip_20201128-134455](https://user-images.githubusercontent.com/24499582/100517087-2c13c480-3180-11eb-8eca-eaa47e1d6f83.png)

**Comment by julonexus** [comment]
There was a workaround about having a registration when no proxy was attached. But it was removed as the process ran fine.
So this case is still to be fixed.

**Comment by DanielLaberge** [comment]
This is still an issue on the latest version (4.2.5), almost one year later.

Is there some workaround?
